ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/location/CaptureLocationTaskMapViewModelTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.google.android.ground.ui.datacollection.tasks.location
+
+import com.google.android.ground.BaseHiltTest
+import com.google.android.ground.ui.common.BaseMapViewModel
+import com.google.common.truth.Truth.assertThat
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import javax.inject.Inject
+
+@HiltAndroidTest
+@RunWith(RobolectricTestRunner::class)
+class CaptureLocationTaskMapViewModelTest : BaseHiltTest() {
+
+  @Inject lateinit var captureLocationTaskMapViewModel: CaptureLocationTaskMapViewModel
+  @Inject lateinit var baseMapViewModel: BaseMapViewModel
+
+  @Test
+  fun testGetMapConfig() {
+    assertThat(captureLocationTaskMapViewModel.mapConfig)
+      .isEqualTo(baseMapViewModel.mapConfig.copy(allowGestures = false))
```

Comment:
Can we create the MapConfig instead of copying from BaseViewModel?

Reply:
Ok, 
The reason I did this way because in viewmodel the original mapConfig is coming from baseViewModel and in viewmodel we copying the data. 

Let's say in future, if something changed in baseViewModel there could be a chance of this test case will fail and we get to know that we have to update it. 

If we make our own mapConfig, it's just testing the copy() is working properly or not. 

Let me know WDYT?

Reply:
By using the `baseMapViewModel` here, we are leaking the implementation details of the class to the test. Even if the value of the base config changes in the future, we should just update the test.

Reply:
Ok, got it